### PR TITLE
Add config option to make test plan optional

### PR DESCRIPTION
**Pull request by orausch**
Test Plan:
make a commit without test plan, run `spr diff` with and
without the new config option.

### CODE REVIEW

**Comment by orausch** [comment]
Very cool tool, thought I'd add this simple option

File: docs/user/commit-message.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ Reviewers: github-username-a, github-username-b
 
 The first line will be the title of the PR created by `spr diff`, and the rest of the lines except for the `Reviewers` line will be the PR description (i.e. the content of the first comment). The GitHub users named on the `Reviewers` line will be added to the PR as reviewers.
 
-The `Test Plan` section is required to be present; `spr diff` will fail with an error if it isn't.
+The `Test Plan` section is required to be present by default; `spr diff` will fail with an error if it isn't.
+You can disable this in the [configuration](/docs/reference/configuration.md).
```
Comment (sven-of-cord) [inline docs/user/commit-message.md:21]:
```suggestion
You can disable this in the [configuration](../reference/configuration.md).
```

The absolute path doesn't work with mdbook (which generates https://getcord.github.io./spr). You can test it yourself, if you like: `cargo install mdbook && mdbook serve` (from the root directory of the repository)

**Review by sven-of-cord — CHANGES_REQUESTED** [review]
Nice work!

Thank you so much and congratulations on becoming the first external code contributor to spr!

It's a shame you can't use `spr` to submit this PR - yet. I hope in the future (once we close #80) that will be possible.

If you can fix the one issue I flagged, we can get this merged.

**Review by sven-of-cord — APPROVED** [review]
Awesome! Thanks again!

Merging in 3... 2... 1...

**Pull request merged**